README.md

```diff
@@ -198,7 +199,105 @@ The library adds a `form-submitted` to the submitted form. You can use it to sty
   }
 }
 ```
+- `controlErrorComponent` - Optional. Allows changing the default component that is used to reflect 
+  the errors. This component should implement the `ControlErrorComponent` interface.  Defaults to
+  `DefaultControlErrorComponent`.
+
+  For example:
+  ```ts
+  // Custom error component that will replace the standard DefaultControlErrorComponent.
```

Comment:
It can be at the description above and not as comment:

```
A common example is when using Ionic. The example below shows
a custom error component that its text wrapped in a `<ion-item>` which best serves a typical Ionic form layout where each form field is enclosed in an `<ion-item>`, all of which are wrapped in an <ion-list>.
```

Reply:
Noted.